docs/README.md

```diff
@@ -5,56 +5,52 @@
 Vue InstantSearch is the ultimate toolbox for creating instant-search
 experiences using [Vue.js](https://vuejs.org/) and [Algolia](https://www.algolia.com/).
 
-## Algolia demo credentials
-
-To be able to use Vue InstantSearch you will need an Algolia account. Once you have an Algolia account, you can get
-the credentials from the [Algolia Dashboard](https://www.algolia.com/api-keys).
-
-For the sake of simplicity, here are some demo credentials that will give you access to
-an e-commerce website's index.
-
- - `appId`: `latency`
- - `searchKey`: `3d9875e51fbd20c7754e65422f7ce5e1`
- - `indexName`: `bestbuy`
-
 ## Setup a new Vue project
 
-You can bootstrap a new Vue project with the [official CLI tool](https://vuejs.org/v2/guide/installation.html#CLI).
+We will use the official [vue-cli](https://vuejs.org/v2/guide/installation.html#CLI) to bootstrap a new Vue project, along with the [webpack-simple template](https://github.com/vuejs/vue-cli#official-templates).
 
 ```sh
 npm install --global vue-cli
+vue init webpack-simple vue-instantsearch-getting-started
+# Default settings are enough, hit `Enter ⏎` at every question
 ```
 
-Now kick off your demo with the `webpack-simple` template:
-
-```sh
-vue init webpack-simple
-```
-
-Finally, you should install the npm dependencies:
+Then install the dependencies of your new project:
 
 ```sh
+cd vue-instantsearch-getting-started
 npm install
 ```
 
 ## Install `vue-instantsearch`
 
-You should now add Vue InstantSearch as a dependency.
+Add Vue InstantSearch as a dependency, it's published on [npm](https://www.npmjs.com):
```

Comment:
I believe this is a good example of a sentence change that looks good to me. Let me know.